### ISSUE TITLE
Incompatible with Apoli

### ISSUE THREAD

**Issue by jennykitten78** [Corail31/tombstone_lite#134]
### Minecraft Version

1.19.2

### Forge Version

43.2.0

### Corail Tombstone Version

8.2.4

### What happened?

Was trying to add Origins to All the Mods 8 and found it (Origins) had an issue with Tombstone. Decided to spin up a test copy using just Tombstone and Origins to see if the error still occurred, which it did. 

[latest.log](https://gist.github.com/jennykitten78/a4d2c25d1dcf52976e35042d08dded90)

[ crash-2023-02-01_12.10.02-client.txt](https://gist.github.com/jennykitten78/db4036e0439ffc4af1d46f29f1259fd9)

This is the results of the test run. Same error i was getting when trying to run it as part of ATM8. I have also posted this to the Origins Forge issues just in case.

Thank you for taking the time to look at this issue report!

### Gametype

Singleplayer

### Happen by testing with only Corail Tombstone (required)

Yes

### Occurence

occurs all the time

### Other relevant mods

Origins (Forge), Origins Classes (Forge)

### Tested without any Forge "alternatives" (Magma, Mohist, and so on)

- [X] Yes

### Valid Launcher

- [X] Yes

**Comment by Corail31** [comment]
Hi Jenny, what is the relation between apoli and Origin ?
You said that you tested with only my mod and origin but there's "apoli".
NB: you have well done to report the issue.
```
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Redirector hasStatusEffectProxy(Lnet/minecraft/world/entity/LivingEntity;Lnet/minecraft/world/effect/MobEffect;)Z in apoli.mixins.json:BackgroundRendererMixin failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap apoli.refmap.json

[12:10:02] [Render thread/WARN]: @Redirect conflict. Skipping apoli.mixins.json:BackgroundRendererMixin->@Redirect::hasStatusEffectProxy(Lnet/minecraft/world/entity/LivingEntity;Lnet/minecraft/world/effect/MobEffect;)Z with priority 1000, already redirected by tombstone.mixins.json:FogRendererMixin->@Redirect::methodHasEffect(Lnet/minecraft/world/entity/LivingEntity;Lnet/minecraft/world/effect/MobEffect;)Z with priority 1000
```

**Comment by jennykitten78** [comment]
Apoli is the name of the library included in Origins as far as I know. Also used/included with Origins Classes Forge

**Comment by Corail31** [comment]
In fact, it's the other mod that is crashing because they didn't add a param require = 0 to their mixin.
https://github.com/EdwinMindcraft/apoli/blob/1.19.x/forge/src/main/java/io/github/apace100/apoli/mixin/BackgroundRendererMixin.java#L26
I can also fix it by changing the priority of my mixin to a lower value.

**Comment by jennykitten78** [comment]
Sorry, I didn't see that Classes was pulled in when I hit install on Origins. Glad you found it though. Really sorry I didn't catch that sooner.

**Comment by Corail31** [comment]
@jennykitten78 i published a new version for 1.19.2 no more using mixin for this part, and resolving this issue.

**Comment by jennykitten78** [comment]
Thank you so much!
